### PR TITLE
Handle mid-cycle candidates who have transferred committees

### PR DESCRIPTION
**Pull request by patphongs**
## Summary

- Addresses https://github.com/fecgov/openFEC/issues/4292
_Creates a way to handle mid-cycle candidates who have converted committees._

## Impacted areas of the application

List general components of the application that this PR will affect:

-  Candidate profile pages for: H0CA08069, H0IL01178, P00010298, P00009092, H8CA25074, H0CA04167
- Committee profile pages for: C00698894, C00697128, C00697441, C00693044, C00634212, C00691790

### Candidate profile page
Screen Shot 2020-06-10 at 4 43 19 PM

### Committee profile page
Screen Shot 2020-06-10 at 9 26 03 AM

## How to test

- checkout this branch
- Look at one of the candidate profile pages listed in the impacted areas section above. See if missing financials message is present: http://localhost:8000/data/candidate/P00010298/
- Look at one of the committee profile pages listed in the impacted areas section above. See if former candidate financials message is present: http://localhost:8000/data/committee/C00697441/
- Look at another random 2020 candidate page with no financials to make sure this message does not appear there. 
- Look at another random 2020 candidate committee page with no financials to make sure the this message does not appear there.

____

### CODE REVIEW

File: fec/data/templates/partials/candidate/financial-summary.jinja [hunk 1]
```diff
@@ -61,7 +62,11 @@
       </div>
     {% else %}
     <div class="entity__figure entity__figure--narrow">
-      {{ missing.missing_financials(name, cycle) }}
+      {% if candidate_id == 'P00010298' %}
```
Comment (lbeaufort) [inline fec/data/templates/partials/candidate/financial-summary.jinja:65]: Let's move this logic to the view

Comment (patphongs) [reply fec/data/templates/partials/candidate/financial-summary.jinja:65]: Thanks @lbeaufort, I was thinking about doing this eventually, especially when we have a list of candidates that meet this criteria

**Review by JonellaCulmer — APPROVED** [review]
Looks great. Thanks @patphongs

**Review by lbeaufort — APPROVED** [review]
@patphongs great work! There are a few areas where the code could be cleaned up, but I think we should 1) add automated tests before refactoring and 2) incorporate the data from the API once it's there.

**Pull request merged**